management/commands/map_places.py

```diff
@@ -1,6 +1,6 @@
 import json
 import logging
-import urllib
+import urllib.request, urllib.parse, urllib.error
```

Comment:
do we technically need urllib.error?  maybe it doesn't matter that an extra thing is being imported...

Reply:
I have no idea, I just do whatever Python's `2to3` tool tells me to do....

I suspect we don't need it, but as it's Python I doubt there's any major downside to an unused import.  If it compiled everything into a local binary that'd be different, but alas we are in Python, not Go :P